network/im/discord-canary/pspec.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+<!DOCTYPE PISI SYSTEM "https://solus-project.com/standard/pisi-spec.dtd">
+<PISI>
+    <Source>
+        <Name>discord-canary</Name>
+        <Packager>
+            <Name>QORTEC</Name>
+            <Email>qortec.ca@gmail.com</Email>
+        </Packager>
+        <Summary>All-in-one voice and text chat for gamers</Summary>
+        <Description>All-in-one voice and text chat for gamers that's free, secure, and works on both your desktop and phone.</Description>
+        <License>Copyright (c) Hammer &amp; Chisel</License>
+        <Archive sha1sum="89adc84b61823c6edc46df939726108780ef2d90" type="binary">https://dl-canary.discordapp.net/apps/linux/0.0.8/discord-canary-0.0.8.deb</Archive>
+        <BuildDependencies>
+            <Dependency>binutils</Dependency>
+        </BuildDependencies>
+    </Source>
+
+    <Package>
+        <RuntimeDependencies>
+            <Dependency>gconf</Dependency>
+        </RuntimeDependencies>
+        <Name>discord-canary</Name>
+        <Icon>discord-canary</Icon>
+        <Files>
+            <Path fileType="executable">/usr/bin</Path>
+            <Path fileType="data">/usr/share/applications</Path>
+            <Path fileType="data">/usr/share/discord-canary</Path>
+            <Path fileType="doc">/usr/share/doc</Path>
+            <Path fileType="data">/usr/share/lintian</Path>
```

Comment:
If there is a issue is with line 30 ``/usr/share/lintian`` then I do not know how to resolve the issue.

When I was looking how to handle **lintian** I came across **n1** project in the git.solus-project.com repository, and they also use the same path for  **lintian**  I use.  - [n1 pspec_x86_64.xml](https://git.solus-project.com/packages/n1/tree/pspec_x86_64.xml)

I asked @JoshStrobl about the **lintian** path on IRC and to my understanding of what he said; it is a valid platform independent path.

I don't know how to proceed, any information would be helpful.